app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -114,6 +117,26 @@ def new_pins_match(self, user_input):
             pins_match = True
         return pins_match
 
+    def authorize_pin(self, pin):
+        authorized = False
+        if self.user.failed_pin_attempts is None:
+            self.user.failed_pin_attempts = 0
+        if self.user.failed_pin_attempts == 3:
```

Comment:
this is probably a ridiculous comment, but I always feel safer when these sorts of checks use `>=` inequality check, so that if somehow the failed attempt count gets to 4, an attacker doesn't then find themsleves with unlimited attempts available.